datasets/xtreme_s/xtreme_s.py

```diff
@@ -0,0 +1,911 @@
+
+_ALL_LANG = set(_BABEL_LANG + _MLS_LANG + _VOXPOPULI_LANG + _COVOST2_LANG + _FLORES_LANG + _MINDS_14_LANG)
+
+_ALL_DATASET_CONFIGS = {
+    "babel": _BABEL_LANG,
+    "mls": _MLS_LANG,
+    "voxpopuli": _VOXPOPULI_LANG,
+    "covost2": _COVOST2_LANG,
+    "fleurs": _FLORES_LANG,
+    "minds14": _MINDS_14_LANG,
+}
+
+# _ALL_LANG = ["ar", "as", "ca", "cs", "cy", "da", "de", "en", "en", "en", "en", "es", "et", "fa", "fi", "fr", "hr", "hu", "id", "it", "ja", "ka", "ko", "lo", "lt", "lv", "mn", "nl", "pl", "pt", "ro", "ru", "sk", "sl", "sv", "sw", "ta", "tl", "tr", "zh"]
+
+_ALL_CONFIGS = []  # e.g. mls.en, covost.en.sv, ...
+for sub_data, langs in _ALL_DATASET_CONFIGS.items():
+    for lang in langs:
+        _ALL_CONFIGS.append(f"{sub_data}.{lang}")
+
+# add fleurs all for langID task
+_ALL_CONFIGS.append("fleurs.all")
+
+
+_DESCRIPTIONS = {  # TOOD(PVP)
+    "babel": "",
+    "mls": """\
+Multilingual LibriSpeech (MLS) dataset is a large multilingual corpus suitable for speech research. The dataset is derived from read audiobooks from LibriVox and consists of 8 languages - English, German, Dutch, Spanish, French, Italian, Portuguese, Polish.
+""",
+    "voxpopuli": "",
+    "covost2": "",
+    "fleurs": "",
+    "minds14": "",
+}
+
+_CITATIONS = {  # TOOD(PVP)
+    "babel": "",
+    "mls": """\
+@article{Pratap2020MLSAL,
+  title={MLS: A Large-Scale Multilingual Dataset for Speech Research},
+  author={Vineel Pratap and Qiantong Xu and Anuroop Sriram and Gabriel Synnaeve and Ronan Collobert},
+  journal={ArXiv},
+  year={2020},
+  volume={abs/2012.03411}
+}
+""",
+    "voxpopuli": "",
+    "covost2": "",
+    "fleurs": "",
+    "minds14": "",
+}
+
+_HOMEPAGE_URLS = {  # TOOD(PVP)
+    "babel": "",
+    "mls": "http://www.openslr.org/94",
+    "voxpopuli": "",
+    "covost2": "",
+    "fleurs": "",
+    "minds14": "",
+}
+
+_VOXPOPULI_AUDIO_URLS = ['https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2009.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2010.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2011.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2012.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2013.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2014.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2015.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2016.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2017.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2018.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2019.tar', 'https://dl.fbaipublicfiles.com/voxpopuli/audios/original_2020.tar']
+
+_DATA_URLS = {  # TODO(PVP)
```

Comment:
Need "fleurs" here too, otherwise the script fails during init :)

Reply:
Yeah, it's not really public yet, so it's only defined in the dataset script on the Hub yet